### PR TITLE
ECAL : New method for MC production : outliers treatment.  Legacy Run2.

### PR DESCRIPTION
**Pull request by depasse**
Use data Pedestals for MC production, correcting outliers by rings to Mean rings.
Legacy Run2 : all the G1/G6 files now completed.

### CODE REVIEW

**Comment by ggovi** [comment]
please test

File: CondTools/Ecal/src/EcalPedestalsHandler.cc [hunk 1]
```diff
@@ -28,35 +28,39 @@ popcon::EcalPedestalsHandler::EcalPedestalsHandler(const edm::ParameterSet & ps)
         m_runtag=ps.getParameter<std::string>("RunTag");
 	m_filename = ps.getUntrackedParameter<std::string>("filename","EcalPedestals.txt");
 	m_runtype = ps.getUntrackedParameter<int>("RunType",1);
+        m_corrected = ps.getUntrackedParameter<bool>("corrected",false);
 
-	std::cout << m_sid<<"/"<<m_user<<"/"<<m_pass<<"/"<<m_location<<"/"<<m_gentag   << std::endl;
+	edm::LogInfo("EcalPedestalsHandler")<<m_sid<<"/"<<m_user<<"/"<<m_pass<<"/"<<m_location<<"/"<<m_gentag;
```
Comment (ggovi) [inline CondTools/Ecal/src/EcalPedestalsHandler.cc]: Please remove the printout of the password

**Comment by boudoul** [comment]
Hello @depasse , could you please edit the title and mention the word "ecal" somewhere in the title, so that it is clear in the release notes that this not a generic infrastructure PR (or we no do wonder if this is for tracker, hcal etc.. ) thank you .

**Comment by ggovi** [comment]
please test

**Comment by ggovi** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**